concepts/05 UI Components/PivotGrid/00 Getting Started with PivotGrid/08 Configure Fields and Areas.md

```diff
@@ -0,0 +1,135 @@
+To display data in the PivotGrid, specify the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array. Each object in it configures a single pivot grid field. Specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to populate the pivot grid field with data.
```

Comment:
> To display data in the PivotGrid, specify the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array. Each object in it configures a single pivot grid field.

"A single" looks like a tautology. It reads as an extra emphasis on the "singularity" aspect:

> ... configures **just only one** pivot grid field.

Just "one" is more than enough here.

```suggestion
To display data in the PivotGrid, specify the [fields[]](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) array. Each object in it configures one grid field. Specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) property to populate the pivot grid field with data.
```

Reply:
The point is that indefinite article already states "one" or "single". If you put "single", it places an extra emphasis on the singularity aspect. I only suggested "one" because it can replace the indefinite article while preserving your emphasis to some extent. Either way, I attracted your attention towards this fact, you can do with it whatever you deem necessary.